### DISCUSSION TITLE
Outbound firewall policy . What ports do I need open?

### DISCUSSION THREAD

**Discussion by pmisner** [FreshRSS/FreshRSS#6021] [Q&A]
Hi, I have a instance of Freshrss hosted on a Linode. I was trying to firewall it off. I'm having a problem with outbound connections. I opened 443/80/53 for HTTPS, HTML, and DNS. When I block all other ports, I find that I'm not receiving feeds, I get the following errors.

  2024-01-03 14:50:15	cURL error 28: Resolving timed out after 20000 milliseconds [https://microsoftsecurityinsights.com/rss]
⚠️	2024-01-03 14:50:15	cURL error 28: Resolving timed out after 20000 milliseconds [https://status.proton.me/history.rss]
⚠️	2024-01-03 14:50:15	cURL error 28: Resolving timed out after 20000 milliseconds [https://www.askwoody.com/feed/]
⚠️	2024-01-03 14:50:15	cURL error 28: Resolving timed out after 20000 milliseconds [https://www.bleepingcomputer.com/feed/]
⚠️	2024-01-03 14:50:15	cURL error 28: Resolving timed out after 20000 milliseconds [https://cyberscoop.com/feed/]

Am I missing a needed port, or is there some other configuration issue. When I allow all outbound packets, it works fine.

Thank you in advance.

**Comment by Frenzie** [comment]
My first instinct is that you may not be allowing those ports for the right process (e.g. PHP CLI for cron), or something similar like allowing them for UDP but not TCP/IP or vice versa.
